subsys/mpsl/fem/Kconfig

```diff
@@ -64,11 +64,27 @@ if MPSL_FEM_NRF21540_GPIO
 
 config MPSL_FEM_NRF21540_TX_GAIN_DB
 	int "TX gain of the nRF21540 PA amplifier in dB"
+	default MPSL_FEM_NRF21540_TX_GAIN_DB_POUTB_PROD
+	help
+	  With the GPIO implementation, the Kconfig can be set either to 20 or 10 dBm.
+	  Using an unsupported value results in build assertion.
+
+	  The default value of 10 dBm is based on nRF21540 Product Specification (v1.0)
+	  and it corresponds to the configuration in which the pin MODE=1 and pin POUTB_SEL=0.
```

Comment:
nit: neither `POUTB_SEL` nor `POUTA_SEL` are pins. We can correct this documentation if we're changing it anyway